aspnetcore/security/enforcing-ssl.md

```diff
@@ -231,55 +229,13 @@ dotnet dev-certs https --help
 
 <a name="trust-ff"></a>
 
-### Trust the HTTPS certificate with Firefox to prevent SEC_ERROR_INADEQUATE_KEY_USAGE error
```

Comment:
To confirm: these are the windows and macos instructions?  If so, I agree that removing them makes sense.